versions.json

```diff
@@ -81,14 +81,6 @@
     "alpine-default": "alpine3.13",
```

Comment:
Hi, I'm currently working on the #1599 PR which implement a new minimal variant for alpine (with no npm nor yarn).
I was noticed that alpine 3.11 will be dropped soon so I wanted to align with this.

While reviewing this PR, I noticed that you upgraded node v12 and v14 to alpine 3.14 but not node v16.
Is this expected or forgotten?

Reply:
@nodejs/docker should we just bump this all up to 3.14? I don't think I touched this one, but might make more sense to make them all the latest